source/sites/publicodes/LandingContent.tsx

```diff
@@ -0,0 +1,17 @@
+export default ({ children, background = false, footer = false }) => (
```

Comment:
**nitpick**: I deduce the naming of the path `source/sites/publicodes` is a legacy from the initial fork of `mon-entreprise` however, I don't understand why we keep it.. Is there a reason?

Reply:
no

Reply:
we should just remove it all at once...